packages/graphql/tests/integration/aggregations/where/AND-OR-operations.int.test.ts

```diff
@@ -174,274 +174,4 @@ describe("Nested within AND/OR", () => {
             ],
         });
     });
-
-    test("AND within an AND", async () => {
-        const query = `
-            query {
-                ${postType.plural}(where: { 
-                    likesAggregate: {
-                        AND: [
-                            { count_LTE: 2 }
-                            {
-                                AND: [
-                                    {
-                                        node: {
-                                            testString_SHORTEST_LT: 4
-                                        }
-                                    }
-                                    {
-                                        node: {
-                                            testString_EQUAL: "${testString5}"
-                                        }
-                                    }
-                                ]
-                            }
-                        ]
-                    }
-                }) {
-                    content
-                }
-            }
-        `;
-
-        const result = await testHelper.executeGraphQL(query);
-
-        expect(result.errors).toBeFalsy();
-        expect(result.data).toEqual({
-            [postType.plural]: expect.toIncludeSameMembers([
-                {
-                    content: content5,
-                },
-            ]),
-        });
-    });
-
-    test("AND within an AND with NOT", async () => {
```

Comment:
Same as before, I know it could sound superfluous, but these tests were very helpful during the refactor.